src/api/test-controller/index.js

```diff
@@ -209,6 +212,51 @@ export default class TestController {
         return this._enqueueCommand('dragToElement', DragToElementCommand, { selector, destinationSelector, options });
     }
 
+    _parseSelectorForScroll (args) {
```

Comment:
You retrieve the selector from arguments, so the `_getSelectorForScroll` is a more suitable name.

Reply:
agree